targets/xbmc

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh -e
```

Comment:
It looks like in the help case, we set `TARGETS='help'`, on line 309 we set `TARGETNOINSTALL='y'` and then source the target file.  So it looks like the right tweak would be actually in [the target common file](https://github.com/dnschneid/crouton/blob/master/targets/common#L29), line 29, where it prints out the help text.  If DESCRIPTION is blank, we don't want to do that.